pkgs/by-name/sq/sqlc/package.nix

```diff
@@ -44,7 +41,7 @@ buildGoModule {
   passthru.tests.version = testers.testVersion {
     package = sqlc;
     command = "sqlc version";
-    version = "v${version}";
+    version = "v${finalAttrs.version}";
```

Comment:
Can you please switch to versionCheckHook ? Have a look at Typst for some inspiration